runtime/onert/core/src/exec/MinMaxData.cc

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "MinMaxData.h"
+
+#include <iostream>
+
+namespace onert
+{
+namespace exec
+{
+
+RawMinMaxDumper::RawMinMaxDumper(const std::string &filename) : _filename(filename) {}
+
+void RawMinMaxDumper::dump(const exec::IOMinMaxMap &input_minmax,
+                           const exec::OpMinMaxMap &op_minmax) const
+{
+  // Find file is already exist for modifying
+  auto file = std::fopen(_filename.c_str(), "rb+");
+  uint32_t runs = 1;
+  if (!file)
+  {
+    // If file is not exist, create new file
+    file = std::fopen(_filename.c_str(), "wb+");
+
+    // Write run count
+    std::fwrite(&runs, sizeof(uint32_t), 1, file);
+  }
+  else
+  {
+    // Read run count
+    auto read_size = std::fread(&runs, sizeof(uint32_t), 1, file);
+    if (read_size != 1)
+      throw std::runtime_error{"Failed to read run count"};
+    runs++;
+
+    // Overwrite run count
+    std::fseek(file, 0, SEEK_SET);
+    std::fwrite(&runs, sizeof(uint32_t), 1, file);
+
+    // Go to end of file to append new data
+    std::fseek(file, 0, SEEK_END);
```

Comment:
Does it update the runs_count in front of the file and append other data to the file at the end?

Reply:
Right. First 4 bit is run count, and append each run's data. So we need to update run count when we append data.